### PR TITLE
Bugfix/FOUR-4840: List of Activities does not mention conditional Event

### PR DESCRIPTION
**Pull request by agustinbusso**
## Issue & Reproduction Steps
When you have a process with an Intermediate Conditional Catch Event message for that event is not showed in list of activities for request.
- Create a process with an Intermediate Conditional Catch Event.
- Create new request and then go to Requests > Select the request and take a look at the list of activities, you should not see a message for the conditional event.

## Solution
- Added message for Conditional Event Definition Interface.
- Added missing translations

## How to Test
- Create a process with an Intermediate Conditional Catch Event.
- Create new request and then go to Requests > Select the request and take a look at the list of activities, you should see a message for the conditional event.

**Sample process**
[Verify Event Based Gateway Conditional and Message Event.json.zip](https://github.com/ProcessMaker/processmaker/files/7677380/Verify.Event.Based.Gateway.Conditional.and.Message.Event.json.zip)

**Working video**

https://user-images.githubusercontent.com/90727999/145242930-5d9a1d39-9180-4d6b-9ffa-10fde2f220b9.mov

## Related Tickets & Packages
- [FOUR-4840](https://processmaker.atlassian.net/browse/FOUR-4840)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-4840-processmaker.ci.processmaker.net

File: ProcessMaker/Listeners/BpmnSubscriber.php [hunk 1]
```diff
@@ -168,7 +169,8 @@ public function onIntermediateCatchEventActivated(IntermediateCatchEventInterfac
     {
         $messages = [
             MessageEventDefinitionInterface::class => 'System is waiting to receive message ":event"',
-            TimerEventDefinitionInterface::class => 'System has scheduled timer: ":event"'
+            TimerEventDefinitionInterface::class => 'System has scheduled timer: ":event"',
```
Comment (danloa) [inline ProcessMaker/Listeners/BpmnSubscriber.php:172]: Correct the text to : 
'System has _**a**_ scheduled timer: ":event"'

File: ProcessMaker/Listeners/BpmnSubscriber.php [hunk 2]
```diff
@@ -168,7 +169,8 @@ public function onIntermediateCatchEventActivated(IntermediateCatchEventInterfac
...
+            ConditionalEventDefinitionInterface::class => 'System is waiting for conditional event: ":event"',
```
Comment (danloa) [inline ProcessMaker/Listeners/BpmnSubscriber.php:173]: Correct the text to:
'System is waiting for _**the**_ conditional event: ":event"',

**Review by danloa — CHANGES_REQUESTED** [review]

**Review by danloa — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-4840-processmaker.ci.processmaker.net

**Pull request merged**